hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/protocol/EndpointService.java

```diff
@@ -245,7 +250,13 @@ public void measureAvails(Collection<MeasurementInstance<L, AvailType<L>>> insta
             Driver<L> driver = session.getDriver();
             for (MeasurementInstance<L, AvailType<L>> instance : instances) {
                 AttributeLocation<L> location = instance.getAttributeLocation();
-                Object o = driver.fetchAttribute(location);
+                Object o = null;
+                try (Context timerContext = diagnostics.getRequestTimer().time()) {
```

Comment:
the problem here is you are not just timing the actual execute of the DMR invocation - you are timing everything around it. I ONLY want diagnostics when you make the DMR call - so limit the amount of extra work the timer is including. There is too much else being including in this timer. Just follow the code in fetchAttribute and see all the extra stuff we are timing.